device/arista/x86_64-arista_7060dx5_32/Arista-7060DX5-32-200Gx50-100Gx14/th4-a7060dx5-32.config.bcm

```diff
@@ -0,0 +1,813 @@
+# configuration yaml file
```

Comment:
Is this file shared for all hwskus?   If so, please move it to platform directory and let the sai.profile to reference to the same file.  If not I would recommend to rename the file to different names according to hwskus.

Reply:
It is not the same for all hwskus, so the filenames have been changed to be different.